pkg/features/kube_features.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
+
+import (
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+)
+
+const (
+	// Every feature gate should add method here following this template:
+	//
+	// // owner: @username
+	// // alpha: v1.4
+	// MyFeature() bool
+
+	// owner: @pmorie
+	// alpha: v1.7
+	//
+	// OriginatingIdentity controls whether the controller should include originating identity in the header of requests
+	// sent to brokers
+	OriginatingIdentity utilfeature.Feature = "OriginatingIdentity"
+)
+
+func init() {
+	utilfeature.DefaultFeatureGate.Add(defaultServiceCatalogFeatureGates)
```

Comment:
At some point, we should start encouraging people to create their own `FeatureGate`.  I don't know for sure that everything wires through, but if it does, do that here.